tools/extract_packets.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
I wonder whether we should add a `requirements.txt` in `tools/` (especially now when we use scapy).

Reply:
I added a separate `requirements.txt` in fa57e5d.

Reply:
The pylint check will fail when Scapy is not installed:

https://github.com/Componolit/RecordFlux-specifications/runs/2110974130?check_suite_focus=true

IMHO all checks and tests should run successfully after installing all dependencies defined in the `requirements.txt` in the top-level directory. I therefore tend to revert this change.

Reply:
Now there is a single `requirements.txt` in the top-level directory again.

Reply:
Sorry for the confusion.